src/Core/Layer/Layer.js

```diff
@@ -1,5 +1,59 @@
 import { EventDispatcher } from 'three';
 
+/**
+ * Layer's sequence (index) event
+ * @event Layer#sequence-property-changed
+ * @property sequence-property-changed {string} emit after the index of a layer has changed
```

Comment:
From the example, it looks like the event has no property named `sequence-property-changed`, but 5 properties: `layerId`, `new`, `previous`, `target`, and `type` (and `new.sequence` and `previous.sequence`)

http://usejsdoc.org/tags-event.html